data/hai/hai reveal 4 middle.txt

```diff
@@ -1510,60 +1913,131 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator, who struggles to compose himself suitably. "I think the Council has heard enough. The petitioner is entitled to a closing statement, unless there is objection?"`
-			`	None of those assembled who might object seem to have recovered their composure enough to stand up quickly, and the moderator, perhaps sensing a politic moment to intervene, quickly announces the closing of the interrogation and calls the Council to hear Teeneep again.`
-			`	Teeneep quickly plasters back on her mask of calm as best she is able to and says, "Thank you all for your ears. You are those we trust to decide, but know this: If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed, no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`
+				`	"Excuse me, I need to check out something in the reading room."`
+					goto reading
+			`	She glances at you briefly. "They are saying: Veeheylur and Misanaya fight over Eyrisen." Her tone carries with it a sense of awe. Despite the growing volume in the room she leans in close to you and continues in scarcely more than a whisper, "The Mythos has been invoked."`
+			`	Aimet painfully resumes his seat and another Elder stands. This Elder had been maintaining a fairly neutral demeanor, but appears to gain a sense of himself. He is quickly heralded by the moderator, who struggles to compose suitably. "I think the Council has heard enough. The petitioner is entitled to a closing statement, unless there is objection?"`
+			`	None of those assembled who might object seem to have recovered their composure enough to stand up quickly. The moderator, perhaps sensing a politic moment to intervene, quickly announces the closing of the interrogation. He calls the Council to hear Teeneep again.`
+			`	Teeneep quickly plasters on a mask of calm as best she is able and says, "Thank you all for your ears. You are those we trust to decide, but know this: If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed, no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`
 			`	One Elder cries out, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!"`
```

Comment:
I know it's not a line that is already _directly_ changed by this PR, but seeing as how the subject of this PR is about this mission...
```suggestion
			`	One Elder cries out, "You wouldn't dare!" to which Teeneep immediately responds, "It is already scheduled to happen unless I actively stop it!"`
```